sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/models/BoundingBox.java

```diff
@@ -11,6 +11,10 @@
     private LatLong topLeft;
     private LatLong bottomRight;
 
+    public BoundingBox() {
```

Comment:
I assume this is to get the JSON support working? If so, it's OK. remove the "super()" though. It's called automatically.

Reply:
Yes, this is to get JSON support working, will remove super!